src/pkg/caendr/caendr/services/cloud/analytics.py

```diff
@@ -37,6 +45,7 @@ def get_weekly_visits(SERVICE):
       'reportRequests': [{
         'viewId': '117392266',
         'dateRanges': [{'startDate': '2015-01-01', 'endDate': datetime.now().date().isoformat()}],
+        # 'dateRanges': [{'startDate': '2015-01-01', 'endDate': '2023-03-31'}],
```

Comment:
remove commented code.

Reply:
removed in the commit [4e1e299](https://github.com/AndersenLab/CAENDR/pull/408/commits/4e1e29981e3dad85cd0ec00ef2fbd2610ad5f949)